lib/node.dart

```diff
@@ -42,18 +54,39 @@ base class Node implements SceneGraph {
   }
 
   Node? _parent;
+
+  /// The parent node of this node in the scene graph.
   Node? get parent => _parent;
   bool _isRoot = false;
 
+  /// The collection of vertices, edges, and faces that represent the geometry of this node,
+  /// along with the material that defines the object's appearance.
+  ///
+  /// The mesh defines the shape of the object through its geometry, which is composed of vertices,
+  /// edges, and faces. The mesh may also include a material, which specifies how the surface
+  /// of the object interacts with light, including properties like color, texture, and reflectivity.
```

Comment:
A `Mesh` is actually a list of `MeshPrimitives`, each of which contain a `Geometry` and a corresponding `Material` to apply. So this would actually be a pretty good description of `MeshPrimitive`, but isn't quite accurate for `Mesh`.